recipe/meta.yaml

```diff
@@ -152,6 +154,7 @@ outputs:
         - bwidget
         - tktable
         - tk
+        - tzdata
```

Comment:
We don't necessarily need a minimum version (the one from `src/extra/tzone/zoneinfo.zip:zoneinfo/VERSION`) here, right?
Just making sure.

Reply:
Yeah, that's okay